build-steps/testing/testing.gradle

```diff
@@ -65,13 +64,14 @@ ext {
             result
         }
 
-        additionalJdksToTest.each { jdk ->
+        testJdks().each { jdk ->
```

Comment:
I guess you did this change to not fail when invoking `testJdk9Jre7/8`? If yes, it would be super nice to add it to the commit message :smiley: (I mean, why was this necessary and what did it solve)